### DISCUSSION TITLE
Unable to install due to error

### DISCUSSION THREAD

**Discussion by MVMInc2022** [ExistentialAudio/BlackHole#727] [General]
### Please Confirm

- [X] I have read the **[FAQ](https://github.com/ExistentialAudio/BlackHole#faq) and [Wiki](https://github.com/ExistentialAudio/BlackHole/wiki)** where most common issues can be resolved
- [X] I have searched **[Discussions](https://github.com/ExistentialAudio/BlackHole/discussions)** to see if the same question has already been asked
- [X] This is a **bug** and not a question about audio routing or configuration, which should be posted in [Discussions](https://github.com/ExistentialAudio/BlackHole/discussions)

### macOS Version

macOS 13 Ventura

### BlackHole Build(s) Affected

- [X] 2 channel
- [ ] 16 channel
- [ ] 64 channel
- [ ] other/custom build

### Describe the bug

When downloading the software I got an error message saying unable to install contact software manufacturer for assistance.

### Reproduction Steps

1.
2.
3.
etc...

### Expected Behavior

I expect the software to be installed

### Screenshots

<img width="1440" alt="Screenshot 2023-05-09 at 2 32 00 PM" src="https://github.com/ExistentialAudio/BlackHole/assets/110066448/9f0e5b0f-e1af-4b80-bff5-b1f77ab26a49">

**Comment by devinroth** [comment]
Did you read the FAQ?

**Comment by devinroth** [comment]
https://github.com/ExistentialAudio/BlackHole#why-is-the-installer-failing

**Comment by sandrocarval** [comment]
Workaround did not work for me. I am running Sonoma 14.4 on a M3. Moved to desktop, documents, home folder, back to downloads folder, same issue. Tested 2ch and 16ch (0.5.0) installers.
